lib/src/base-styles/_nx-page-header.scss

```diff
@@ -41,61 +41,60 @@ $main-header-product-grid-rows: $main-header-branding-height / 2;
   color: $nx-off-white;
   display: -ms-grid;
   display: grid;
-  grid-column-gap: 2px;
-  grid-row-gap: 2px;
-  -ms-grid-rows: $main-header-product-grid-rows 2px auto;
-  grid-template-rows: $main-header-product-grid-rows auto;
-  -ms-grid-columns: $main-header-branding-height 2px auto;
-  grid-template-columns: $main-header-branding-height auto;
+  grid-column-gap: $nx-spacing-xxs;
+  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
+  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
+  grid-template-columns: $main-header-product-height auto;
   grid-template-areas:
     "logo title"
     "logo version";
-  height: $main-header-branding-height;
+  height: $main-header-product-height;
   margin-top: 10px;
```

Comment:
Use a variable?

Reply:
> Use a standard variable?

Neither 12px nor 8px looked right.

Reply:
> Neither 12px nor 8px looked right.

Hmm, maybe margin isn't the right solution here anyway then.  How about `align-self: center`?

Reply:
> > Neither 12px nor 8px looked right.
> 
> Hmm, maybe margin isn't the right solution here anyway then. How about `align-self: center`?

Ah, yes, that is a much better solution: https://github.com/sonatype/sonatype-react-shared-components/pull/13/commits/dbffb953b81bcfb70bf8e102cdeda9ecc7a59067